pallets/parachain-staking/src/weights.rs

```diff
@@ -725,30 +818,59 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+		//  Measured:  `0 + x * (56 ±0) + y * (21 ±0) + z * (41 ±0)`
+		//  Estimated: `38555 + x * (142 ±0) + y * (191 ±0) + z * (291 ±0)`
+		// Minimum execution time: 110_244_000 picoseconds.
+		Weight::from_parts(82_475_228, 38555)
+			// Standard Error: 9_271
+			.saturating_add(Weight::from_parts(37_524, 0).saturating_mul(x.into()))
+			// Standard Error: 9_271
+			.saturating_add(Weight::from_parts(59_736, 0).saturating_mul(y.into()))
+			// Standard Error: 32_357
+			.saturating_add(Weight::from_parts(209_896, 0).saturating_mul(z.into()))
 			.saturating_add(T::DbWeight::get().reads(8_u64))
 			.saturating_add(T::DbWeight::get().writes(8_u64))
-			.saturating_add(Weight::from_parts(0, 145).saturating_mul(x.into()))
-			.saturating_add(Weight::from_parts(0, 132).saturating_mul(y.into()))
-			.saturating_add(Weight::from_parts(0, 307).saturating_mul(z.into()))
+			.saturating_add(Weight::from_parts(0, 142).saturating_mul(x.into()))
+			.saturating_add(Weight::from_parts(0, 191).saturating_mul(y.into()))
+			.saturating_add(Weight::from_parts(0, 291).saturating_mul(z.into()))
+	}
+	/// Storage: System Account (r:2 w:2)
+	/// Proof: System Account (max_values: None, max_size: Some(116), added: 2591, mode: MaxEncodedLen)
+	/// Storage: ParachainStaking DelegatorState (r:2 w:2)
+	/// Proof Skipped: ParachainStaking DelegatorState (max_values: None, max_size: None, mode: Measured)
+	/// Storage: ParachainStaking CandidateInfo (r:1 w:1)
+	/// Proof Skipped: ParachainStaking CandidateInfo (max_values: None, max_size: None, mode: Measured)
+	/// Storage: ParachainStaking AutoCompoundingDelegations (r:1 w:1)
+	/// Proof Skipped: ParachainStaking AutoCompoundingDelegations (max_values: None, max_size: None, mode: Measured)
+	/// Storage: ParachainStaking TopDelegations (r:1 w:1)
+	/// Proof Skipped: ParachainStaking TopDelegations (max_values: None, max_size: None, mode: Measured)
+	/// Storage: ParachainStaking BottomDelegations (r:1 w:1)
+	/// Proof Skipped: ParachainStaking BottomDelegations (max_values: None, max_size: None, mode: Measured)
+	/// Storage: Balances Locks (r:2 w:2)
+	/// Proof: Balances Locks (max_values: None, max_size: Some(1287), added: 3762, mode: MaxEncodedLen)
+	/// Storage: ParachainStaking DelegationScheduledRequests (r:1 w:1)
+	/// Proof Skipped: ParachainStaking DelegationScheduledRequests (max_values: None, max_size: None, mode: Measured)
+	/// Storage: ParachainStaking CandidatePool (r:1 w:1)
+	/// Proof Skipped: ParachainStaking CandidatePool (max_values: Some(1), max_size: None, mode: Measured)
+	/// Storage: ParachainStaking Total (r:1 w:1)
+	/// Proof Skipped: ParachainStaking Total (max_values: Some(1), max_size: None, mode: Measured)
+	fn delegate_with_auto_compound_worst() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `48167`
+		//  Estimated: `426257`
+		// Minimum execution time: 275_279_000 picoseconds.
+		Weight::from_parts(275_279_000, 426257)
```

Comment:
significant for delegate weight. May not fit in a block.

Reply:
As per ts-tests we are now able to fit at most 9 delegate calls (was ~50 before).

Reply:
I think this is true only for the worstcase scenario, which test ?